### PR TITLE
fix: when tenant is deleted,remove namespaces of unsupported types from DB

### PR DESCRIPTION
**Pull request by MatousJobanek**
When a tenant is going to be deleted, then remove also DB records of namespaces of old unsupported namespaces (run, stage, jenkins) to not fail when checking if all namespaces were removed.

### CODE REVIEW

File: openshift/action_test.go [hunk 1]
```diff
@@ -562,6 +562,21 @@ func (s *ActionTestSuite) TestDeleteAction() {
 		assertion.AssertTenant(t, repo).DoesNotExist()
 	})
 
+	s.T().Run("ManageAndUpdateResults should delete also unsupported entities", func(t *testing.T) {
```
Comment (xcoulon) [inline openshift/action_test.go]: did you mean "unsupported namespaces" ?

File: controller/tenants_test.go [hunk 2]
```diff
@@ -190,6 +190,34 @@ func (s *TenantsControllerTestSuite) TestSuccessfullyDeleteTenants() {
...
+			SetMatcher(test.ExpectRequest(test.HasJWTWithSub("devtools-sre"))).
+			Reply(200).
+			BodyString(`{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Success"}`)
+		gock.New(test.ClusterURL).
+			Delete("/oapi/v1/projects/foo").
+			SetMatcher(test.ExpectRequest(test.HasJWTWithSub("devtools-sre"))).
+			Reply(200).
+			BodyString(`{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Success"}`)
+
+		fxt := tf.FillDB(t, s.DB, tf.AddSpecificTenants(tf.SingleWithName("foo")),
```
Comment (xcoulon) [inline controller/tenants_test.go]: ok, so in this test, you fill the DB with records for some `user`, `che`, `jenkins`, `stage` and `run` tenants but only `user` and `che` will be deleted? All other namespaces/tenants are assumed to be already deleted (ie, `jenkins`, `stage` and `run`)? And at the end of the test, all records were removed from the DB?

File: openshift/action_test.go (hunk 1 above)
Comment (MatousJobanek) [reply openshift/action_test.go]: yup :-) fixed in 4a7f712

File: controller/tenants_test.go (hunk 2 above)
Comment (MatousJobanek) [reply controller/tenants_test.go]: Yes, exactly :-)

Comment (xcoulon) [reply controller/tenants_test.go]: ok, thanks for clarifying ;)

File: openshift/action_test.go (hunk 1 above)
Comment (xcoulon) [reply openshift/action_test.go]: thanks ;)

**Pull request merged**